providerframework/metaservice/lib/go/client.go

```diff
@@ -15,10 +15,25 @@
 package metaservice
 
 import (
+	"net"
+
 	"google.golang.org/grpc"
 )
 
-func NewLBDial(target string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
+func newLBDial(target string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
```

Comment:
This function should be exported as it is used outside the package

Reply:
updated code as per comment